packages/mantine/src/components/table/TablePagination.tsx

```diff
@@ -8,11 +8,19 @@ interface TablePaginationProps {
      * The total number of page. Use null only if your table is paginated client side
      */
     totalPages: number | null;
+    /**
+     * The callback if the current page is changed.
+     *
+     * @param pageSize The number if item per page.
+     * @param pageIndex The index of the updated page.
+     */
+    onPageChange?: (pageSize: number, pageIndex: number) => void;
```

Comment:
A bit picky, but since the order of parameters in this function is important, and given its name `onPageChange`, it makes more sense to have the page index first.